distribution/tools/upgrade-cli/src/main/java/org/opensearch/upgrade/ImportLog4jPropertiesTask.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.upgrade;
+
+import org.opensearch.cli.Terminal;
+import org.opensearch.common.collect.Tuple;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+
+/**
+ * Imports Log4j properties from an existing elasticsearch installation.
+ */
+class ImportLog4jPropertiesTask implements UpgradeTask {
```

Comment:
This needs to be updated to only import any additional settings and not copy and overwrite the entire file.